Automattic/wp-calypso
# Stats: Apply extended quantity to the last tier

Related to #85109

## Proposed Changes

* Restore the slider view for a single tier by hiding the slider and only showing the price and quantity labels
* Fix the default slider quantity when not interacting with the slider (previously, lack of interaction could result in the quantity set to 0)
* Applying double quantity for the last tier instead of `null` when redirecting to checkout

## Testing Instructions



* create a testing blog and navigate to a live branch with `stats/tier-upgrade-slider` feature flag
* purchase final commercial tier
* navigate to the purchase page again
* verify that the slider is not visible, but the labels with the price and quantity are visible
* purchase the final tier
* quantity is set to `2 000 000` and you might see it in the URL, but I'm investigating why it doesn't apply the proper limit after purchase 

## Pre-merge Checklist
